README.rst

```diff
@@ -528,6 +528,41 @@ Example:
       ]
 
 
+Portlets
+~~~~~~~~
+
+Portlets can easily be defined by using the ``_portlets`` key, expecting a dict (portlet_manager : configuration).
+It is possible to define the the context portlets (``portlets``) and set the inheritance settings (``blacklist_status``).
```

Comment:
1) duplicate `the` => "define the ~the~ context portlets"

2) I thinkt "create assignments" is more accurate than "create portlets"; a developer creates a portlet by coding it, but the user just creates a portlet assignment in the database.
I personally would rename the `portlets` key to `assignments`.
Therefore => `It is possible to define the context portlet assignments (``assignments``) ....`